modules/sandboxed-containers-deploying-osc-with-other-ocp-components.adoc

```diff
@@ -0,0 +1,16 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/understanding_sandboxed_containers.adoc
+
+:_content-type: CONCEPT
+[id="sandboxed-containers-with-other-ocp-components_{context}"]
```

Comment:
@bobfuru I know. It's prep for the future, since there will be additional components added to this over time. I can change it if you think I need to, but I'l have to change it again in the future to be more general. WDYT?